--- libnifalcon-1.0.2/CMakeLists.txt.orig	2010-11-27 05:42:07.000000000 +0100
+++ libnifalcon-1.0.2/CMakeLists.txt	2014-12-31 15:20:50.459408548 +0100
@@ -81,14 +81,13 @@
 
 #Find boost, used all over the place
 SET(Boost_USE_MULTITHREADED ON)
-SET(Boost_USE_STATIC_LIBS ON)
 
 FIND_PACKAGE(Boost REQUIRED)
 IF(NOT Boost_FOUND)
   MESSAGE(FATAL_ERROR "libnifalcon requires a minimum of the Boost 1.33 headers to build")
 ENDIF(NOT Boost_FOUND)
 
-FIND_PACKAGE(Boost COMPONENTS program_options thread)
+FIND_PACKAGE(Boost COMPONENTS program_options thread system)
 
 ######################################################################################
 # Project specific globals
--- libnifalcon-1.0.2/examples/CMakeLists.txt.orig	2010-11-27 05:42:07.000000000 +0100
+++ libnifalcon-1.0.2/examples/CMakeLists.txt	2014-12-31 15:58:29.952647059 +0100
@@ -128,7 +128,7 @@
   IF(X11_FOUND AND X11_XTest_LIB)
 	INCLUDE_DIRECTORIES(${X11_INCLUDE_DIR})
 	SET(MOUSE_SRCS falcon_mouse/falcon_mouse.cpp falcon_mouse/falcon_mouse_x11.cpp)      
-	SET(MOUSE_LINK_LIBS ${LIBNIFALCON_EXE_PO_LINK_LIBS} ${X11_LIBRARIES} ${X11_XTest_LIB}) 
+	SET(MOUSE_LINK_LIBS ${LIBNIFALCON_EXE_PO_LINK_LIBS} ${X11_LIBRARIES} ${X11_XTest_LIB} boost_system) 
   ELSE(X11_FOUND AND X11_XTest_LIB)
 	MESSAGE(STATUS "Not compiling falcon_mouse example - X11 compiling requires the XTest library")
   ENDIF(X11_FOUND AND X11_XTest_LIB)
